CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md

```diff
@@ -245,3 +245,31 @@ You can show a bottom navigation bar by passing true to `GiniConfiguration.share
 your own by implementing the `ReviewScreenBottomNavigationBarAdapter` interface and passing it to `GiniConfiguration.shared.reviewNavigationBarBottomAdapter`.
 
 You can find more details [here](https://www.figma.com/file/1985HMF83siAXmysSn3dC6/iOS-Gini-Capture-SDK-3.0.0-UI-Customisation?node-id=261%3A8256&t=6sAk7LGf1mi3zV9L-1).
+
+## No results screen
+
+The new no results screen uses the global UI customization options. You can discard the old screen specific
+customizations.
+
+### Breaking Changes
+
+#### Removed localization keys:
+
+`ginicapture.noresults.warning`
+`ginicapture.noresults.collection.header`
+`ginicapture.noresults.gotocamera`
+`ginicapture.noresults.warningHelpMenu`
+
+### New features
+
+#### New localization keys:
+
+`ginicapture.noresult.enterManually`
+`ginicapture.noresult.retakeImages`
+
+#### Bottom navigation bar
+
+You can show a bottom navigation bar by passing true to `GiniConfiguration.shared.bottomNavigationBarEnabled`. There is a default implementation, but you can also use
+your own by implementing the `NoResultBottomNavigationBarAdapter` interface and passing it to `GiniConfiguration.shared.noResultNavigationBarBottomAdapter`.
+
+You can find more details [here](https://www.figma.com/file/1985HMF83siAXmysSn3dC6/iOS-Gini-Capture-SDK-3.0.0-UI-Customisation?node-id=263%3A6989&t=7wXW9XyhTUcmp5sk-1).
```

Comment:
[Suggestion] I think we share the link for the customization guide - similar to the error screen:

 You can find more details [here](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/3.0.0-beta02/features.html#error-screen-customization) and [here]((https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/3.0.0-beta02/customization-guide.html#error-screen).